src/Perpetuum/Services/Weather/WeatherInfo.cs

```diff
@@ -1,19 +1,20 @@
 using System;
 using System.Text;
 using Perpetuum.Timers;
+using Perpetuum.Zones;
 
-namespace Perpetuum.Zones
+namespace Perpetuum.Services.Weather
 {
     public class WeatherInfo
     {
-        private readonly int _current;
+        public int Current { get; private set; }
         public int Next { get; private set; }
```

Comment:
Given that these are only set by the constructor you can drop the `private set;` part to make them readonly properties.